scss/variables/modal.js

```diff
@@ -0,0 +1,75 @@
+( function( root, factory ) {
+	if ( typeof define === "function" && define.amd ) {
+		define( [ "./chassis", "./colors", "./typograpy" ], factory );
+	} else if ( typeof exports === "object" ) {
+		require( "./chassis" );
+		require( "./colors" );
+		module.exports = factory( require( "./typography" ) );
+	} else {
+		root.chassis = factory( root.chassis );
+	}
+}( this, function( chassis ) {
+
+chassis.modal = {
+
+	"shadows": {
+		name: "Modal shadow styles",
+		value: {
+			"base": "0px 8px 11px 0px rgba(0,0,0,0.15)"
+		}
+	},
+	"borders": {
+		name: "Modal border styles",
+		value: {
+			"default": "1px solid " + chassis.colors.grey.value.base,
+			"radius": "3px"
+		}
+	},
+	"padding": {
+		name: "Modal Padding value",
+		value: {
+			"padding": "15px"
+		}
+	},
+	"element": {
+		name: "Generic modal styles",
+		value: {
+			"padding": "0px",
+			"margin": "10px",
+			"background": chassis.colors.background.value.base
+		}
+	},
+	"title": {
+		name: "Modal Title Styles",
+		value: {
+			"font-size": "24px",
+			"font-weight": "700",
+			"margin": "0"
```

Comment:
Executing em function (https://github.com/jquery/css-chassis/blob/master/scss/atoms/typography/_functions.scss) on 0 values without a unit results in a warning "Warning: 0em/px isn't a valid CSS value."